src/routes/analytics/DashboardPage.tsx

```diff
@@ -19,6 +20,11 @@ export function DashboardPage(props: Props) {
         dashboardId={dashboardId}
         withTitle
         withDownloads={false}
+        renderDrillThroughQuestion={() => (
+          <Container size="1000px" w="100%">
+            <InteractiveQuestionView isSaveEnabled={false} />
+          </Container>
+        )}
```

Comment:
Uses the themed drill-throughs instead of the default views, which looks weird on Acme theme.

<img src="https://github.com/user-attachments/assets/5dc9e971-6efd-42fc-9903-6c8bb627080d" width="350">